### ISSUE TITLE
Wildcard content types support

### ISSUE THREAD

**Issue by jordandobrev** [cdimascio/express-openapi-validator#270]
Using content types `*.*` does not work atm when specified in requestBody.

```
requestBody:
        content:
          "*/*":
            schema:
```

When route is reached the validation returns `unsupported media type`

**Comment by cdimascio** [comment]
Thanks for the issue @jordandobrev. Would you be up for taking a stab at it? 

Here are some tips:

I believe [this method](https://github.com/cdimascio/express-openapi-validator/blob/53e53d772865d2adedeef89d7702909ebf6e9c8f/src/middlewares/parsers/body.parse.ts#L28) needs to be updated to handle wildcards. That is, it should select a matching wildcard schema if a match is not found. 

You could add a test case for your scenario [here](https://github.com/cdimascio/express-openapi-validator/blob/master/test/request.bodies.ref.spec.ts#L44) and put the openapi bits [here](https://github.com/cdimascio/express-openapi-validator/blob/master/test/resources/request.bodies.ref.yaml#L10)

Note, I've set up Gitpod for this project, so get a full dev environment set up in [one click](https://gitpod.io/#https://github.com/cdimascio/express-openapi-validator). 

Basically, click the link above, once it completes type `npm t` to run the tests. I've also available on our gitter [channel](https://gitter.im/cdimascio-oss/community)

**Comment by jordandobrev** [comment]
Yeah sure I’ll have a go this or next week and will PR it.

**Comment by cdimascio** [comment]
Awesome! Thanks so much.

**Comment by cdimascio** [comment]
available in `v3.12.0`

**Comment by cdimascio** [comment]
@all-contributors add @jordandobrev for code and test
